src/codeu/chat/server/Model.java

```diff
@@ -134,6 +142,24 @@ public void add(Message message) {
   public StoreAccessor<String, Message> messageByText() {
     return messageByText;
   }
+<<<<<<< HEAD
+
+  public void changeAuthority(ConversationHeader conversation, Uuid targetUser, byte authorityByte)
+  {
+    LinkedHashMap<Uuid,Byte> userAuthority;
+    if(authority.get(conversation.id) == null)
+    {
+       authority.put(conversation.id, new LinkedHashMap<>());
+    }
+    userAuthority = authority.get(conversation.id);
+    userAuthority.put(targetUser, authorityByte);
+  }
+
+  public boolean validateAuthority(ConversationHeader conversation, Uuid targetUser, byte authorityByte)
+  {
+    LinkedHashMap<Uuid, Byte> userAuthority = authority.get(conversation.id);
+    return userAuthority.get(conversation.id) != null && userAuthority.get(targetUser).byteValue() == authorityByte;
+=======
```

Comment:
Where's the } in this method?? For next time, know that if someone forgets a semicolon or one of these>> ] } ) it's a sign that they didn't compile their code before committing. I understand if you're just pushing these incomplete things now, but making this kind of mistake in the code review phase (which we're not in, technically this is a ghost review) would be a big no-no, and by then you'd need to compile anyway.